<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyConfig</class>
 <widget class="QDialog" name="KeyConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_left" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>124</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_left">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>J</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>左移</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_right" native="true">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>9</y>
     <width>123</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_right">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>L</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>右移</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_left_3" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>59</y>
     <width>124</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_left_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>N</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>左3</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_right_3" native="true">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>59</y>
     <width>123</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_right_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>I</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>右3</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_left_direct" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>109</y>
     <width>124</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_left_direct">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>U</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>左头</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_right_direct" native="true">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>109</y>
     <width>123</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_right_direct">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>O</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>右头</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_down" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>159</y>
     <width>124</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_down">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>K</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>下落</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_bottom" native="true">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>159</y>
     <width>123</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_bottom">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>space</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>到底</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_turn" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>209</y>
     <width>124</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_turn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>转-顺</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_turn_anti" native="true">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>209</y>
     <width>123</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_turn_anti">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>21</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>43</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>转-逆</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
